lib/dkdeploy/tasks/db.rake

```diff
@@ -146,7 +146,7 @@ namespace :db do
         execute :rm, '-f', remote_zipped_dump_file
 
         ignore_tables_command_line = ignore_tables.inject('') do |command_line, table|
-          command_line << " --ignore-table=#{db_settings.fetch('name')}.#{table}"
+          command_line + " --ignore-table=#{db_settings.fetch('name')}.#{table}"
```

Comment:
This should be `+=`, I think.

Reply:
No

```
rubocop
Inspecting 42 files
..........................W...............

Offenses:

lib/dkdeploy/tasks/db.rake:149:11: W: Lint/UselessAssignment: Useless assignment to variable - command_line. Use + instead of +=. (https://github.com/rubocop-hq/ruby-style-guide#underscore-unused-vars)
          command_line += " --ignore-table=#{db_settings.fetch('name')}.#{table}"
          ^^^^^^^^^^^^
```